src/feed/LoginFeed.ts

```diff
@@ -61,13 +64,41 @@ export default function useLoginFeed() {
         }
 
         if ("Notification" in window && Notification.permission === "granted") {
-            for (let nx in notifications.filter(a => (a.created_at * 1000) > readNotifications)) {
-                //let n = new Notification(`New reply!`, { body: nx.content, icon: Nostrich });
-                //console.log(n);
+            for (let nx of notifications.filter(a => (a.created_at * 1000) > readNotifications)) {
+                sendNotification(nx)
+                    .catch(console.warn);
             }
         }
         dispatch(addNotifications(notifications));
         db.users.bulkPut(profiles);
         dispatch(addDirectMessage(dms));
     }, [main]);
+}
+
+async function makeNotification(ev: TaggedRawEvent) {
+    switch (ev.kind) {
+        case EventKind.TextNote: {
+            let from = await db.users.get(ev.pubkey);
+            let name = getDisplayName(from, ev.pubkey);
+            return {
+                title: `Reply from ${name}`,
+                body: ev.content.substring(0, 50)
+            }
+        }
+    }
+    return null;
+}
+
+async function sendNotification(ev: TaggedRawEvent) {
+    let n = await makeNotification(ev);
+    if (n != null && Notification.permission === "granted") {
+        let worker = await navigator.serviceWorker.ready;
+        worker.showNotification(n.title, {
+            body: n.body,
```

Comment:
We need to at least extract mentions to #e and #p in body or they appear as raw `#[0]` index notation

Reply:
![image](https://user-images.githubusercontent.com/1172179/213464083-28f939e2-3ad7-4f93-8bd7-6a9342ff1d16.png)